tests/nested/core/core20-kernel-failover/task.yaml

```diff
@@ -31,7 +31,7 @@ execute: |
   echo "Wait for snapd to be available"
   retry=120
   wait=1
-  until nested_exec command -v snap; do
+  until tests.nested exec command -v snap; do
```

Comment:
probably a TODO, but this is part of the regular core vm bringup so we most likely don't nned it anymore. IIRC the test was written before we added those helpers.